api/users.go

```diff
@@ -10,21 +9,15 @@ import (
 
 	"github.com/vocdoni/saas-backend/db"
 	"github.com/vocdoni/saas-backend/internal"
-	"github.com/vocdoni/saas-backend/notifications"
 	"go.vocdoni.io/dvote/log"
 	"go.vocdoni.io/dvote/util"
 )
 
-// sendUserCode method allows to send a code to the user via email or SMS. It
-// generates a verification code and stores it in the database associated to
-// the user email. If the mail service is available, it sends the verification
-// code via email. If the SMS service is available, it sends the verification
-// code via SMS. The code is generated associated a the type of code received,
-// that can be either a verification code or a password reset code. Other types
-// of codes can be added in the future. If neither the mail service nor the SMS
-// service are available, the verification code will be empty but stored in the
-// database to mock the verification process in any case.
-func (a *API) sendUserCode(ctx context.Context, user *db.User, t db.CodeType) error {
+// generateVerificationCode method generates and stores in the database a new
+// verification code for the user and verification code type provided. Both
+// parameters are required. It returns the generated verification code and an
+// error if the verification code could not be generated or stored in the database.
+func (a *API) generateVerificationCode(user *db.User, codeType db.CodeType) (string, error) {
```

Comment:
Not sure why you removed the notification sending from this function, since you actually do quite similar notification handling in all the 3 places where this function is being used. Is it to avoid passing arguments?

Reply:
To support HTML notification templates also for admin invitations in `api/organizations.go`, the verification code logic does not apply there and is replaced by the invitation code logic.